content/compose/compose-file/05-services.md

```diff
@@ -932,7 +932,9 @@ been the case if `group_add` were not declared.
 
 ## healthcheck
 
-{{< include "compose/services-healthcheck.md" >}}
+{{< include "compose/services-healthcheck.md" >}} 
+
+For more information on `HEALTHCHECK`, see the [Dockerfile reference](../../reference/dockerfile.md#healthcheck).
```

Comment:
I would also add in this file references for the for top-level links for networks, volumes and  secrets

Reply:
and merge [multiple Compose files](https://docs.docker.com/compose/multiple-compose-files/).